pkg/rpc/adminservice/named_entity.go

```diff
@@ -0,0 +1,78 @@
+package adminservice
+
+import (
+	"context"
+
+	"github.com/lyft/flyteadmin/pkg/rpc/adminservice/util"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/core"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func (m *AdminService) GetNamedEntity(ctx context.Context, request *admin.NamedEntityGetRequest) (*admin.NamedEntity, error) {
+	defer m.interceptPanic(ctx, request)
+	if request == nil {
+		return nil, status.Errorf(codes.InvalidArgument, "Incorrect request, nil requests not allowed")
+	}
+
+	// We require a valid resource type to retrieve metadata
+	if request.ResourceType == core.ResourceType_UNSPECIFIED {
```

Comment:
can you move this into the validation in the manager method to be consistent with other request validation? the only reason other adminservice/ wrappers inject the ResourceType for other requests is because the url references the entity type even though the request body doesn't specify the ResourceType

Reply:
Sure. Another case of following/copying code from another handler

Reply:
Oh actually, the manager already validates the resource type. I'm just going to remove this.